client/landing/stepper/declarative-flow/internals/steps-repository/completing-purchase/index.tsx

```diff
@@ -53,11 +54,15 @@ const CompletingPurchase: Step = function CompletingPurchase( { navigation, flow
 			blogdescription: state.siteDescription,
 		} );
 
+		if ( flow && [ 'newsletter', 'link-in-bio' ].includes( flow ) ) {
```

Comment:
It might be nice if to import the array of launchpad flows rather than hard coding. That way we don't have to remember to add them here when we add each new flow. There should be a source of truth where we define the potential flows. 

Looks like we may be (kind of) doing that here: 
https://github.com/Automattic/wp-calypso/blob/trunk/client/landing/stepper/declarative-flow/internals/steps-repository/launchpad/tasks.tsx

Reply:
I think it's a bit too confusing to use keys of `launchpadFlowTasks` as flow types.

Reply:
I think it's a bit too confusing to use keys of `launchpadFlowTasks` as flow types.

Reply:
I think that could make sense as the source of truth. That is where the task lists are defined and without a task list we have no real launchpad flow.

Perhaps we should set up an export from there, reflecting the flows that are defined. Then we can follow up to import that here and in other places to use in place of hardcoding `[ 'newsletter', 'link-in-bio' ]`.

Reply:
> I think it's a bit too confusing to use keys of launchpadFlowTasks as flow types.

oops, missed this comment before my last response. But i agree! Importing and using keys would be awkward, instead we should probably set up an export from that file that is just a simple array reflecting those keys. This can be a follow up and not a blocker for this PR in any way though.